arm-recoveryservicesbackup/readme.md

```diff
@@ -31,20 +31,24 @@ directive:
 
 ### C#
 
-```yaml
```

Comment:
Looks like this is from this pull request - https://github.com/Azure/azure-rest-api-specs/pull/1289/files Maybe when i synced to the latest, it added it to the commit files.